engine/src/main/scala/cromwell/engine/workflow/lifecycle/execution/EngineJobExecutionActor.scala

```diff
@@ -144,9 +148,10 @@ class EngineJobExecutionActor(replyTo: ActorRef,
 
   when(BackendIsCopyingCachedOutputs) {
     // It's possible that cache writing was turned off due to a `HashError`, hence the `writeToCache` guard.
-    case Event(response: SucceededResponse, data @ ResponsePendingData(_, _, Some(Success(hashes)))) if effectiveCallCachingMode.writeToCache =>
-      saveCacheResults(hashes, response.updatedData(data))
-    case Event(response: SucceededResponse, data @ ResponsePendingData(_, _, None)) if effectiveCallCachingMode.writeToCache =>
+//    case Event(response: SucceededResponse, data @ ResponsePendingData(_, _, Some(Success(hashes)), _)) if effectiveCallCachingMode.writeToCache =>
+//      saveCacheResults(hashes, response.updatedData(data))
+    //Ruchi: No need for a state to re-save outputs that are saved in the first place--two copies of the output for the same task?
```

Comment:
The previous code was saving the full set of hashes, which may be a superset of what was used to determine the cache hit.  This is intended for future-proofing in case we ever expand the set of keys that are taken into consideration for the cache hit.